op-scim-bridge/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
```

Comment:
This line has changed. We need to use apiVersion: v2 for helm3. The template files refer to the kubernetes API version, so those are fine at v1.